x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/input/http/HttpInputTests.java

```diff
@@ -267,7 +267,7 @@ public void testThatExpectedContentTypeOverridesReturnedContentType() throws Exc
         ExecutableHttpInput input = new ExecutableHttpInput(httpInput, httpClient, templateEngine);
 
         Map<String, String[]> headers = new HashMap<>(1);
-        String contentType = randomFrom("application/json", "application/json; charset=UTF-8", "text/html", "application/yaml",
+        String contentType = randomFrom("application/json", "application/json;charset=utf-8", "text/html", "application/yaml",
```

Comment:
just to make it consistent with other usages of charset in our codebase, but it actually is not relevant to the PR